widgets/src/main/resources/org/pentaho/mantle/public/messages/mantleMessages_CN.properties

```diff
@@ -18,7 +18,7 @@ workspace=\u5de5\u4f5c\u5340
 myDesktop=\u6211\u7684\u684c\u9762
 about=\u95dc\u65bc...
 aboutDialogTitle=\u95dc\u65bc Pentaho \u4f7f\u7528\u8005\u4e3b\u63a7\u53f0
-logoPanelWebsite= 
+logoPanelWebsite=

```

Comment:
These empty messages are supposed to be empty and were just changed because of applying formatting? just wanted to check if they are intentionally blank. I don't think you're using them.